.docker-setup.sh

```diff
@@ -24,7 +24,6 @@ missingModules=""
 #Check everything that needs to be in the $PATH is in there.
 #Bash doesn't let this work if this is in an if statement for some reason it has to be chained
 type -P "docker" &>/dev/null && echo "docker found..." || missingModules="${missingModules} docker"
-type -P "docker-compose" &>/dev/null && echo "docker-compose found..." || missingModules="${missingModules} docker-compose"
```

Comment:
I can understand why this might be problematic / scary. The reason I filed this PR is because on a modern release of RHEL we must use backports / unstable downloads to get an unsupported version of `docker-compose` -- happy to discuss alternatives.